modules/virt-configuring-workload-update-methods.adoc

```diff
@@ -41,10 +41,16 @@ spec:
     batchEvictionSize: 10 <4>
     batchEvictionInterval: "1m0s" <5>
 ----
```

Comment:
Per our current guidelines, should there be an ellipses at the end of YAML snippet to indicate that the file continues?